fix(docs): Updated development documentation. (#55)
patternfly/react-user-feedback: packages/module/patternfly-docs/content/examples/basic.md

import { FeedbackModal } from "@patternfly/react-user-feedback";
import feedbackImage from '@patternfly/react-user-feedback/dist/esm/images/rh_feedback.svg';

> The user feedback extension is a PatternFly React based component used to collect user feedback.  Examples of how to use this extension are documented below.  This extensions allows users to submit feedback, report a bug, request support, as well as join a mailing list to stay updated on the latest news and research opportunities.

Author: Done.

## Examples

> ### Basic example

Author: Done.

> This is an example of the minimum needed to create a feedback modal.  A user can create a feedback modal that just contains an option to provide feedback.

Author: I made the URL example the basic example now.

> This is an example of the minimum needed to create a feedback modal.  A user can create a feedback modal that just contains an option to provide feedback.

Reviewer: Could you keep the original basic example, but instead just make the "Share feedback" item point to the "Share feedback" form (the one used in the advanced example)? That way the basic example only has the "bare minimum" item.

If you can do that, then I think we could still remove the "URL example", and just mention that you can point to URLs in the advanced example. I think that seems like the most clear and concise way of accurately conveying all of the functionality.

Does that make sense?

```js file="./Basic.tsx"

```

### URL example

> Developers have the option to use URLs instead of the built in forms to collect data.  This is useful if you have a custom form that you want to use to collect data.  The URL will be opened in a new tab.

Author: Done.

```js file="./URL.tsx"

```

> ### Advanced feedback modal

Author: Done.

> This modal is an example of the using the built in forms to collect data.

Author: Done.

```js file="./Advanced.tsx"

```

> ### Advanced with email passed to modal

Author: Done.

> A developer can also pass in the email address for a user to the feedback modal.  This is useful if you have a user's email address and want to pre-populate the email field in the feedback modal.

Reviewer: To automatically pass a user's email address into a feedback modal, use the `email` property. This allows you to pre-populate the modal's email field.

Author: Done.

```js file="./AdvancedWithEmail.tsx"

```

> ### Async example  

Author: Done.

> The user feedback extension supports the ability to make aynchronous calls to a backend service.  This is useful if you want to send the data collected in the feedback modal to a backend service.  The example below shows how to use the async function to send the data to a backend service.

Reviewer: ```suggestion
User feedback supports the ability to make asynchronous calls to send data collected from the modal to a backend service. 

The following example demonstrates how to use the `async` function to send data to a backend service.
```

Author: Done.

```js file="./Async.tsx"

```

> ### i18n

Author: yeah.

> Developers can change what is displayed in the feedback modal by passing in a custom i18n object.  This is useful if you want to change the text that is displayed in the feedback modal.

Reviewer: ```suggestion
To customize the content displayed in the feedback modal, pass in a custom `i18n` object with values for each property of`<FeedbackModal>`. This allows you to prepare a modal for different languages and requirements.
```
